README.md

```diff
@@ -56,7 +56,7 @@ jobs:
 
 - `arch` – target architecture
   - native compilation:
-    - `x64` (default) or its synonyms: `amd64`, `win64`
+    - `x64` (default) or its synonyms: `amd64`, `win64`, `x86_64`
```

Comment:
Well, you forgot "x86-64".

Reply:
Right 👍 Thanks. I've added it to the README as well (without a PR, minor doc change after all).

It seems we've opened a hole to hell with these aliases...

Reply:
Yes.  
After a month, you may forget the difference between "x86_64", "x86_amd64" and "amd64" ;)